modules/develop-for-operators.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * architecture/understanding-openshift-development.adoc
+
+[id="develop-for-operators_{context}"]
+= Develop for Operators
+
+Packaging and deploying your application as an Operator is a particularly good idea if you are offering that application to be run by others. As noted earlier, Operators add a lifecycle component to your application that acknowledges that the job of running an application is not done when it is installed.
```

Comment:
I think we need more info about Operators and a link about making them. There might be a decent one in the 4.x collection.